stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb

```diff
@@ -291,61 +516,16 @@ public postfix func -- (x: inout CGFloat) -> CGFloat {
   return tmp 
```

Comment:
Remove `++` and `--`?  Seems like an incomplete implementation of one of the older proposals.